scripts/latest-revision.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# finds out the commit sha of the latest revision of a repository,
+# with minimal cloning
+set -e
+
+if [ "$#" -ne 1 ]; then
+    echo "USAGE $0 GIT_REMOTE_REPOSITORY"
+    exit 1
+fi
+
+remote="$1"
+folder=$(basename "$remote")
```

Comment:
I'm not suggesting you change it, but `basename \/` => `/` which then becomes `rm -rf "/"` two lines later.

Reply:
Let's do some check here to avoid corner cases